pkg/landscaper/controllers/execution/controller.go

```diff
@@ -65,7 +65,14 @@ func (c *controller) Reconcile(ctx context.Context, req reconcile.Request) (reco
 	}
 
 	if lsv1alpha1helper.IsCompletedExecutionPhase(exec.Status.Phase) {
-		logger.V(7).Info("execution item is already in completion phase")
+		op := execution.NewOperation(operation.NewOperation(logger, c.client, c.scheme, c.eventRecorder), exec, false)
+		phaseChanged, err := op.HandleDeployItemPhaseChanges(ctx, logger)
+		if err != nil {
+			return reconcile.Result{}, lserrors.NewWrappedError(err, "Reconcile", "HandleDeployItemPhaseChanges", err.Error())
+		}
+		if !phaseChanged {
```

Comment:
why do we need the phase changed here?
just for the info log?

Reply:
Yes, basically that.

Reply:
then we could move the log message to the function directly and do not need to return that value at all?

Reply:
There is a log inside the function, but we have also a log if the phase _didn't_ change (so if basically nothing happened). Should we remove that?

Reply:
I moved the log output inside the function and dropped the bool return value.